.github/workflows/check-pull-request.yml

```diff
@@ -25,3 +25,11 @@ jobs:
       - run: npm run format:check
       - run: npm run lint
       - run: npm test
+
+#     - name: SonarCloud Scan
+#       if: github.actor != 'dependabot[bot]'
+#       uses: SonarSource/sonarcloud-github-action@master
+#       env:
+#         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+#         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+#       continue-on-error: true
```

Comment:
Do we include this line in the template? 
I recommend it because... SoundCloud but maybe some do not want it as default